feat(eslint-plugin-fiori-tools): add "liveMode" rule for ODataV4 applications to display "GO" button in preview (#4851)
SAP/open-ux-tools: packages/eslint-plugin-fiori-tools/README.md

|   Since   | Rule | Description | Recommended | Recommended for S/4HANA |
|:---------:|------|-------------|:-----------:|:-----------------------:|

> |  new      | [sap-no-live-mode](docs/rules/sap-no-live-mode.md) | Ensures that the live mode is not enabled. | | ✅ |

Reviewer: ```suggestion
|  new      | [sap-no-live-mode](docs/rules/sap-no-live-mode.md) | Ensures that live mode is not enabled. | | ✅ |
```

> |  9.13.0   | [sap-description-column-label](docs/rules/sap-description-column-label.md) | Ensures that the description (text) property referenced via `Common.Text` has a meaningful `Common.Label` — not a generic value such as `"Name"` or `"Description"`, and not the same label as the ID property. | | ✅ |

Reviewer: ```suggestion
|  9.13.0   | [sap-description-column-label](docs/rules/sap-description-column-label.md) | Ensures that the description text property referenced using the `Common.Text` annotation has a meaningful `Common.Label` annotation. It must not be a generic value such as "Name" or "Description", and not the same label as the `ID` property. | | ✅ |
```
